packages/components/src/radio/src/Radio.css

```diff
@@ -26,15 +28,24 @@
     display: block;
     background-color: var(--hop-neutral-icon-active);
     border-radius: var(--hop-shape-circle);
-    transform: scale(0);
+    transform: scale(0) translate(-50%, -50%);
+    transform-origin: top left;
     transition: all var(--o-ui-easing-duration-1) var(--o-ui-easing-productive);
     position: absolute;
     top: 50%;
     left: 50%;
     width: var(--o-ui-radio-button-size);
     height: var(--o-ui-radio-button-size);
-    margin-top: calc(var(--o-ui-radio-button-size) * -1 / 2);
-    margin-left: calc(var(--o-ui-radio-button-size) * -1 / 2);
+}
+
+/* BUTTON | SMALL */
+.o-ui-radio-sm .o-ui-radio-button {
+    width: 1rem;
+    height: 1rem;
+}
+
+.o-ui-radio-sm .o-ui-radio-button::before {
+    --o-ui-radio-button-size: 0.5rem;
```

Comment:
it's 0.5rem because we want the dot to be 50% of the radio button we could benefit from a calc here or a transform scale